### PR TITLE
Move proxy section

### PR DESCRIPTION
**Pull request by Rookeret73**
#1966

Move proxy section from troubleshooting to setting up
Split rook proxy comments to rook setup pages
Delete agentless page from agent setup
Delete mention of simple https

### CODE REVIEW

File: docs/installation-node.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ __Pre-requisites:__
     ```javascript
     const rook = require('rookout/auto_start');
     ```
+
+### installing the npm package using a proxy
```
Comment (LiranLast) [inline docs/installation-node.md]: This is a minor nuisance - should be marginalized as much as possible

File: docs/installation-java.md [hunk 2]
```diff
@@ -26,6 +26,16 @@ this can be done manually or through the help of a build tool such as [Gradle](h
 Once your application is deployed, navigate to the Rookout App Instances page to make sure it is available for debugging.
 If you encounter any issues, check out our [Troubleshooting section](troubleshooting-rooks.md)
 
+### Rook setup using a proxy
```
Comment (LiranLast) [inline docs/installation-java.md]: This is a minor nuisance - should be marginalized as much as possible

File: docs/installation-python.md [hunk 3]
```diff
@@ -26,6 +26,16 @@ __Pre-requisites:__
     from rook import auto_start
     ```
     
+### Installing the Rookout pypi package using a proxy
```
Comment (LiranLast) [inline docs/installation-python.md]: This is a minor nuisance - should be marginalized as much as possible

Comment (Rookeret73) [reply docs/installation-python.md]: Marginalized proxy in all three rook setup sections

**Pull request merged**